taxonomies/origins.txt

```diff
@@ -24,16 +24,117 @@ synonyms:en:Southwest, South West
 synonyms:en:Southeast, South East
 synonyms:en:Eastern Central, East Central
 synonyms:en:Western Central, West Central
+synonyms:az:Şimal-qərbi, Şimal Qərb
+synonyms:az:Şimal-şərqi, Şimal Şərq
+synonyms:az:Cənub-qərbi, Cənub Qərb
+synonyms:az:Cənub-şərqi, Cənub Şərq
+synonyms:az:Şərq mərkəzi, Şərq Mərkəz
+synonyms:az:Qərb mərkəzi, Qərb Mərkəz
+synonyms:bg:Северозапад, Северен Запад
+synonyms:bg:Североизток, Северен Изток
+synonyms:bg:Югозапад, Южен Запад
+synonyms:bg:Югоизток, Южен Изток
+synonyms:bg:Източно-централен, Източен Централ
+synonyms:bg:Западно-централен, Западен Централ
+synonyms:bn:উত্তরপশ্চিম, উত্তর পশ্চিম
+synonyms:bn:উত্তরপূর্ব, উত্তর পূর্ব
+synonyms:bn:দক্ষিণপশ্চিম, দক্ষিণ পশ্চিম
+synonyms:bn:দক্ষিণপূর্ব, দক্ষিণ পূর্ব
+synonyms:bn:পূর্ব কেন্দ্রীয়, পূর্ব কেন্দ্র
+synonyms:bn:পশ্চিম কেন্দ্রীয়, পশ্চিম কেন্দ্র
+synonyms:ca:Nord-oest, Nord Oest
+synonyms:ca:Nord-est, Nord Est
+synonyms:ca:Sud-oest, Sud Oest
+synonyms:ca:Sud-est, Sud Est
```

Comment:
"PartA-PartB" is same as "PartA PartB", right?
```suggestion
```

Reply:
not sure. @stephanegigandet why did we did that ?

Reply:
@benbenben2 the English version already has those synonyms. So I'd say we keep them for other languages, and remove them if we learn in the future that the parser handles that correctly ?

Reply:
ok, sure